### ISSUE TITLE
Accessibility: Image link is missing alternative text - A 2.4.4 Link Purpose (In Context)

### ISSUE THREAD

**Issue by elrayle** [cul-it/exhibits-library-cornell-edu#481]
### Describe the bug
On home page, all exhibit card images are missing alt text.  They are decoration and should be using the title.

![image](https://user-images.githubusercontent.com/6855473/127207587-3257a663-e91c-45bc-bd0d-6823faea75e5.png)

### Solution

This issue is also described in [Spotlight Issue #2767](https://github.com/projectblacklight/spotlight/issues/2767) and is addressed by
[Spotlight PR #2768](https://github.com/projectblacklight/spotlight/pull/2768) which is pending review.  

Temporarily, the same change is added to our exhibits app as a customization in PR #480. 

Once the Spotlight PR is merged, the file added in the local PR can be deleted.

### To Reproduce
1. Home
2. Run Site Improve
3. See error

**Comment by elrayle** [comment]
The update to NextGen removed this as an error.  Closing all issues and PRs.  Reverted the customization in our code.
